DICOM/src/Microsoft.Health.DeID.SharedLib/Settings/EncryptionSetting.cs

```diff
@@ -0,0 +1,52 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Security.Cryptography;
+using System.Text;
+using Microsoft.Health.Dicom.DeID.SharedLib.Exceptions;
+
+namespace Microsoft.Health.DeID.SharedLib.Settings
+{
+    public class EncryptionSetting
```

Comment:
Seems redact, dateshift, cryptoHash, perturb are all verbs, why we use Encryption here

Reply:
Fixed it.